### PR TITLE
add tooltip

### PR DESCRIPTION
**Pull request by bingzhang**
## Description
_Please provide a summary of the pull request and the issue it fixes. Please add necessary details, context, dependencies, explanation of when review is needed (see next section), etc._

**Fixes #(add issue number here and remove parentheses)**

[comment]: # (This project only accepts pull requests related to open issues.)
[comment]: # (If suggesting a new feature or change, please discuss it in an issue first.)
[comment]: # (If fixing a bug, there should be an issue describing it with steps to reproduce.)

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Immediately
- [ ] Within a week
- [ ] When possible

## Type of changes
_Please select a relevant option:_

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [ ] I have updated the [CHANGELOG](../CHANGELOG.md).
- [ ] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [ ] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [ ] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

---

[comment]: # (Template credit: This pull request template is based on Embedded Artistry {https://github.com/embeddedartistry/templates/blob/master/.github/PULL_REQUEST_TEMPLATE.md}, Clowder {https://github.com/clowder-framework/clowder/blob/develop/.github/PULL_REQUEST_TEMPLATE.md}, and TalAter {https://github.com/TalAter/open-source-templates} templates.)

### CODE REVIEW

**Comment by sandeep-ps** [comment]
Hi, @bingzhang, I wondered if you have explored the Bootstrap tooltips (https://getbootstrap.com/docs/4.0/components/tooltips/) before? It seems like they are much more compact in implementation and come with many configurable options.

**Comment by bingzhang** [comment]
> Hi, @bingzhang, I wondered if you have explored the Bootstrap tooltips (https://getbootstrap.com/docs/4.0/components/tooltips/) before? It seems like they are much more compact in implementation and come with many configurable options.

not yet. We can merge it and do the improvement on it.

**Comment by sandeep-ps** [comment]
FYI, I have linked the relevant issue to this PR.

File: templates/events/add-new-event.html [hunk 1]
```diff
@@ -805,6 +810,30 @@ <h5>Add Sub-Event </h5>
...
+  position: relative;
+  display: inline-block;
+  border-bottom: 1px dotted black;
+}
+
+.mytooltip .mytooltiptext {
+  visibility: hidden;
+  width: 1200px;
+  background-color: black;
+  color: #fff;
```
Comment (sandeep-ps) [inline templates/events/add-new-event.html:823]:
```suggestion
  color: #343a40;
```
Suggesting a color change.

File: templates/events/add-new-event.html [hunk 2]
```diff
@@ -805,6 +810,30 @@ <h5>Add Sub-Event </h5>
...
+  {#text-align: center;#}
+  border-radius: 6px;
+  padding: 10px 0;
```
Comment (sandeep-ps) [inline templates/events/add-new-event.html:826]:
```suggestion
  padding: 10px;
```
I think this is looking better with padding on all sides.

File: templates/events/event-edit.html [hunk 3]
```diff
@@ -33,6 +33,29 @@
...
+  .mytooltip {
+  position: relative;
+  display: inline-block;
+  border-bottom: 1px dotted black;
+}
+
+.mytooltip .mytooltiptext {
+  visibility: hidden;
+  width: 1200px;
+  background-color: black;
```
Comment (sandeep-ps) [inline templates/events/event-edit.html:46]:
```suggestion
  background-color: #343a40;
```

File: templates/events/event-edit.html [hunk 4]
```diff
@@ -33,6 +33,29 @@
...
+  color: #fff;
+  {#text-align: center;#}
+  border-radius: 6px;
+  padding: 10px 0;
```
Comment (sandeep-ps) [inline templates/events/event-edit.html:50]:
```suggestion
  padding: 10px;
```

**Comment by sandeep-ps** [comment]
Hi, Bing, I have suggested a couple of minor changes. The rest generally looks good. 

I think the width of the tooltip probably should not be fixed as that cause the tooltip to cut off as the screen size becomes smaller. Please create an issue for this as a future improvement. Thanks.

**Review by sandeep-ps — APPROVED** [review]

**Pull request merged**